apps/nextjs-website/src/lib/strapi/codecs/ProductCodec.ts

```diff
@@ -57,9 +57,10 @@ export const ProductRelationshipsCodec = t.strict({
   }),
 });
 
-export const BaseProductWithBannerLinksCodec = t.strict({
+export const BaseProductWithRelationsCodec = t.strict({
   attributes: t.intersection([
     BaseProductAttributesCodec,
+    ProductRelationshipsCodec,
```

Comment:
I think we should rename this constant to `ProductRelationsCodec` (relationship tends to be preferred for human connections, relationship for more abstract connections like these)